vllm_spyre/v1/worker/spyre_worker.py

```
@@ -163,6 +171,12 @@ def compile_or_warm_up_model(self) -> None:
                 "[WARMUP] (%d/%d) for prompt length %d, decoding %d tokens "
                 "with batch size %d...", i + 1, num_shape_combinations,
                 prompt_len, num_decode_tokens, batch_size)
+
+            if self.model_runner.is_multimodal():
+                logger.warning(
+                    "[WARMUP] - this model is multimodal, but mm features are "
+                    "not passed through in static batching at the moment; "
+                    "this will probably break compiled models!")
```

I'd just raise a value error here since it's not supported